### PR TITLE
[REF] pos,*: rename all methods with camelCase

### PR DESCRIPTION
**Pull request by sjai-odoo**
In this commit:
-------------------
- Renamed all js methods in point_of_sale and all related modules of point_of_sale to camelCase from snake_case.

Related PR: https://github.com/odoo/enterprise/pull/72769
Task: 4269182

### CODE REVIEW

**Comment by sjai-odoo** [comment]
**_Whenever we are ready to r+ this branch, please inform once as we have to rebase this branch again as there can be high chances of having many conflicts._**

File: addons/sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class ComboConfiguratorDialog extends Component {
         });
         if (this.props.edit) this._initSelectedComboItems();
         this._selectSingleComboItems();
-        this.getPriceUrl = '/sale/combo_configurator/get_price';
+        this.getPriceUrl = '/sale/combo_configurator/getPrice';
```
Comment (chevalierv) [inline addons/sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.js:49]:
```suggestion
        this.getPriceUrl = '/sale/combo_configurator/get_price';
```

File: addons/website_sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ patch(ComboConfiguratorDialog.prototype, {
         super.setup(...arguments);
 
         if (this.props.isFrontend) {
-            this.getPriceUrl = '/website_sale/combo_configurator/get_price';
+            this.getPriceUrl = '/website_sale/combo_configurator/getPrice';
```
Comment (chevalierv) [inline addons/website_sale/static/src/js/combo_configurator_dialog/combo_configurator_dialog.js:22]:
```suggestion
            this.getPriceUrl = '/website_sale/combo_configurator/get_price';
```

**Review by chevalierv — CHANGES_REQUESTED** [review]
Hello @sjai-odoo 

The filter was a bit too large 😄

**Comment by sjai-odoo** [comment]
> Hello @sjai-odoo
> 
> The filter was a bit too large 😄

Hello @chevalierv ,
Yes :sweat_smile:, I have changed it please check it again.

File: odoo/cli/templates/l10n_payroll/models/hr_payslip_worked_days.py.template
Comment (davidmonnom) [inline odoo/cli/templates/l10n_payroll/models/hr_payslip_worked_days.py.template:1]: oops should not be modified

**Review by davidmonnom — CHANGES_REQUESTED** [review]

File: odoo/cli/templates/l10n_payroll/models/hr_payslip_worked_days.py.template
Comment (sjai-odoo) [reply odoo/cli/templates/l10n_payroll/models/hr_payslip_worked_days.py.template:1]: Reverted that change,
Thanks

**Review by davidmonnom — APPROVED** [review]
@robodoo r+

File: addons/point_of_sale/static/src/app/models/pos_order.js [hunk 3]
```diff
@@ -268,22 +268,22 @@ export class PosOrder extends Base {
...
+    isEmpty() {
         return this.lines.length === 0;
     }
 
     updateSavedQuantity() {
         this.lines.forEach((line) => line.updateSavedQuantity());
     }
 
-    assert_editable() {
+    assetEditable() {
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/src/app/models/pos_order.js:279]: @sjai-odoo Can you please correct this ?

Comment (sjai-odoo) [reply addons/point_of_sale/static/src/app/models/pos_order.js:279]: Hi @vlst-odoo 
Thanks, updated [in this pr](https://github.com/odoo/odoo/pull/189565)

**Pull request closed without merging**